examples/plot_intelex.py

```diff
@@ -130,7 +130,7 @@
 log_loss_score = log_loss(y_test, y_proba)
 print(
     f"[sklearnex] Inference took t_opt = {t_opt:.2f}s with a log-loss score of"
-    f" {log_loss_score:.1%}"
+    f" {log_loss_score:.1f}"
```

Comment:
```suggestion
    f" {log_loss_score:.3f}"
```

Same comment as above.